pyproject.toml

```diff
@@ -9,10 +9,9 @@ requires = [
     "setuptools",
     "cmake",
     "typing_extensions",
-    "future",
+    "future ; python_version<3",
```

Comment:
why not delete this entirely?

Reply:
@ezyang https://github.com/pytorch/pytorch/blob/b847ac227f0435b26f7c6f12c685d22d78eec17f/setup.py#L204, although it might come backed into python 2.7 come to think of it so it might be fine to remove entirely

Reply:
yes, just kill the future

Reply:
I think in principle this helps people out if they accidentally call setup.py with a python2 interpreter but just like, whatever, idgaf